tensornetwork/tests/network_test.py

```diff
@@ -230,6 +230,16 @@ def test_outer_product(backend):
   assert d.name == "D"
 
 
+def test_outer_product_without_legs(backend):
+  a = tn.Node(np.ones((2, 4, 5)), name="A", backend=backend)
+  b = tn.Node(np.ones(()), name="B", backend=backend)
+  c = tn.outer_product(a, b, name="C")
+
```

Comment:
perhaps test the other order also the arguments of outer product?

Reply:
+1